classes/teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(name, age)
```

Comment:
- [ ] Kindly pass in the other parameters for creating a person. We need to pass in the name and age.

- [ ] We also need to follow the order of the parameters in the `#super` method.

  ```
  def initialize(specialization, age, name = 'Unknown')
    super(age, name)
  ```